epoxy-processor/src/main/java/com/airbnb/epoxy/ViewAttributeInfo.kt

```diff
@@ -274,7 +298,10 @@ internal class ViewAttributeInfo(
     ) {
         for (annotationMirror in paramElement.annotationMirrors) {
             val annotationElement = types.asElement(annotationMirror.annotationType)
-            val builder = AnnotationSpec.builder(ClassName.get(annotationElement as TypeElement))
+            if (ModelViewProcessor.modelPropAnnotationsNames.contains(ClassName.get(annotationElement as TypeElement))) {
```

Comment:
can do `if (annotationClass in ModelViewProcessor.modelPropAnnotationsNames) continue`